docs/flask.rst

```diff
@@ -0,0 +1,90 @@
+Using WhiteNoise with Flask
+============================
+
+.. note:: To use WhiteNoise with a non-Flask application see the
+   :doc:`generic WSGI documentation <base>`.
+
+This guide walks you through setting up a FLask project with WhiteNoise.
+In most cases it shouldn't take more than a couple of lines of configuration.
+
+I mention Heroku in a few place as that was the initial use case which prompted me
+to create WhiteNoise, but there's nothing Heroku-specific about WhiteNoise and the
+instructions below should apply whatever your hosting platform.
+
+1. Make sure where your *static* is located
+-------------------------------------------
+
+If you're familiar with Flask you'll know what to do. If you're just getting started
+with a new Flask project then the default is the ``static`` folder  in the root path
+of the application.
+
+Check the ``static_folder`` argument in `Flask Application Object documentation <http://flask.pocoo.org/docs/api/#application-object>`_ for further information.
+
+
+
+2. Enable WhiteNoise
+--------------------
+
+In the file where you create your app you instantiate Flask Application Object (the ``flask.Flask()`` object). All you have to do is to wrap it with ``WhiteNoise()`` object.
+
+If you use Flask quick start approach it will look something like that:
+
+
+.. code-block:: python
+
+    from flask import Flask
+    from whitenoise import WhiteNoise
+
+    app = WhiteNoise(Flask(__name__), root='static/')
+
+If you opt for the `pattern of creating your app with a function <http://flask.pocoo.org/snippets/20/>`_, then it would look like that:
+
+.. code-block:: python
+
+    from flask import Flask
+    from sqlalchemy import create_engine
+    from whitenoise import WhiteNoise
+
+    from myapp import config
+    from myapp.views import frontend
+
+    def create_app(database_uri, debug=False):
+        app = Flask(__name__)
+        app.debug = debug
+    
+        # set up your database
+        app.engine = create_engine(database_uri)
+    
+        # add your modules
+        app.register_module(frontend)
```

Comment:
Is `register_module` available in recent versions of Flask? 
I think the current way of doing this would be via `register_blueprint`.